tests/e2e/test_create_project.py

```diff
@@ -21,41 +17,14 @@ def test_e2e_create_project(page: Page, db: DatabaseTestUtils):
     button = page.get_by_role("button", name="Create Project")
     button.click()
 
-    page.wait_for_url("project/1", timeout=1000)
-
     expect(page.get_by_role("heading", name="My new project")).to_be_visible()
 
 
-def test_e2e_create_project_with_tasks(page: Page, db: DatabaseTestUtils):
```

Comment:
Is this test removed intentional or by accident? I see no alternative for this and maybe it was removed due to a rebase conflict?

Reply:
Indeed accidently removed. added again